causely/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0ac3b5eb-46d2-4091-a001-e95dc5782609",
+  "app_id": "causely",
+  "display_on_public_website": false,
```

Comment:
Is this intentionally false?

Reply:
no, it was generated by ddev - changing it